cmd/openshift-install/create.go

```diff
@@ -489,6 +521,11 @@ func logComplete(directory, consoleURL string) error {
 	logrus.Infof("To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=%s'", kubeconfig)
 	logrus.Infof("Access the OpenShift web-console here: %s", consoleURL)
 	logrus.Infof("Login to the console with user: %q, and password: %q", "kubeadmin", pw)
+
+	if err = checkComputeReplicas(config); err != nil {
```

Comment:
don't put this as part of the logComplete. i don't think it belongs here.. we should probably just do that outside this function (before)

Reply:
+1. Moved it into waitForInstallComplete and converted the error into a warning.